contracts/core/tokens/RebalancingSetToken.sol

```diff
@@ -619,4 +621,24 @@ contract RebalancingSetToken is
     {
         return biddingParameters.combinedNextSetUnits;
     }
+
+    /* ============ Getter Functions ============ */
+
+    function clearAuctionState()
+        internal
+    {
+        nextSet = address(0);
+        auctionLibrary = address(0);
+        startingCurrentSetAmount = 0;
+        delete auctionParameters;
+
+        if (rebalanceState == RebalancingHelperLibrary.State.Drawdown) {
+            biddingParameters.minimumBid = 0;
+            biddingParameters.remainingCurrentSets = 0;
+            delete biddingParameters.combinedCurrentUnits;
+            delete biddingParameters.combinedNextSetUnits;
+        } else {
+            delete biddingParameters;    
```

Comment:
What's the difference between the two cases? Don't they do the same thing?

Reply:
We need the `combinedTokenArray` for the `withdrawFromFailedRebalance`

Reply:
Hmm - why don't we delete `biddingParameters` and generate a variable called `withdrawComponents` or something for withdrawing?

Reply:
@asoong What do you think?

Reply:
Yeah not a fan of this, also not a fan of duplicating the state. Ideally we store them separately and only delete `minimumBid` and `remainingCurrentSets` but i'm guessing they're grouped together now so they can be passed around more efficiently

Reply:
Actually, they're only passed together when placing bid and getting bid price